app/forms/apo_form.rb

```diff
@@ -100,16 +98,42 @@ def tags_client
     Dor::Services::Client.object(model.pid).administrative_tags
   end
 
+  LICENSES = {
+    'pddl' => { label: 'Open Data Commons Public Domain Dedication and License 1.0',
+                uri: 'http://opendatacommons.org/licenses/pddl/1.0/' },
+    'odc-by' => { label: 'Open Data Commons Attribution License 1.0',
+                  uri: 'http://opendatacommons.org/licenses/by/1.0/' },
+    'odc-odbl' => { label: 'Open Data Commons Open Database License 1.0',
+                    uri: 'http://opendatacommons.org/licenses/odbl/1.0/' },
+    'by' => { label: 'Attribution 3.0 Unported',
+              uri: 'https://creativecommons.org/licenses/by/3.0/' },
+    'by-sa' => { label: 'Attribution Share Alike 3.0 Unported',
+                 uri: 'https://creativecommons.org/licenses/by-sa/3.0/' },
+    'by_sa' => { label: 'Attribution Share Alike 3.0 Unported',
+                 uri: 'https://creativecommons.org/licenses/by-sa/3.0/',
+                 deprecation_warning: 'license code "by_sa" was a typo in argo, prefer "by-sa"' },
```

Comment:
I know you extracted this from dor-services, but I wonder if we can drop this? Could be a follow-up.

Reply:
That seems good to me. I'll drop that change.

Reply:
Also checked in QA and stage and _nada_. :beers: